Source/Blazorise/wwwroot/fileEdit.js

```diff
@@ -75,7 +75,10 @@ export function open(element, elementId) {
 
 // Reduce to purely serializable data, plus build an index by ID
 function mapElementFilesToFileEntries(element) {
-    element._blazorFilesById = {};
+
+    if (element._blazorFilesById == null) {
```

Comment:
I think it is best to check it as `if (!element._blazorFilesById)` instead of specifically for `null`.

Reply:
yeah.. This is so alien to me, but makes "jsense"...